play/index.js

```diff
@@ -55,6 +55,10 @@ play("Carousel", module)
       h, containerWidth, [h(Carousel, {}, generateSlideImages(h))]
     )
   )
+  .add("too few per page", h => createContainer(
```

Comment:
Do you mind updating this to "Too few per page" just for consistency?

Reply:
Done